delta_shooter/game_mechanics.py

```diff
@@ -106,54 +123,63 @@ def __init__(
         else:
             self.screen = DummyScreen(GAME_SIZE)
 
-        self.reset()
         self.num_envs = 1
-        self.action_space = gym.spaces.Discrete(4)  # type: ignore
-
+        self.observation_space = gym.spaces.Box(low=-1, high=1, shape=(18,))
+        self.action_space = gym.spaces.Discrete(6)  # type: ignore
```

Comment:
Are these used anywhere? Don't seem to be

Reply:
Needed for stable baselines but nothing else can take out